bench/wtperf/wtperf.h

```diff
@@ -251,3 +296,14 @@ void	 lprintf(const CONFIG *, int err, uint32_t, const char *, ...)
 __attribute__((format (printf, 4, 5)))
 #endif
 ;
+
+static inline void
```

Comment:
Thanks. Nope - I just have difficulty with inline semantics and felt like a sanity check was worthwhile.